haxelib/aze/display/TileLayer.hx

```diff
@@ -90,7 +90,11 @@ class TileLayer extends TileGroup
 			if (!child.visible) continue;
 			#end
 			
+			#if (flash||js)

```

Comment:
Apparently this causes problems with neko, that's why it was simplified to a systematic use of Std.is.